src/components/Map/BoothList.tsx

```diff
@@ -0,0 +1,136 @@
+import { NavLink } from 'react-router-dom';
+
+import { styled } from 'styled-components';
+
+import InfoWithIcon from '../MapDetail/InfoWithIcon';
+import Booth from '../../types/Booth';
+
+const Container = styled.div`
+  overflow-y: scroll;
+  width: 100%;
+  height: 80%;
+  margin-bottom: 100px;
+`;
+
+const PreButton = styled(NavLink)`
+  cursor: pointer;
+  text-decoration: none;
+  color: inherit;
+`;
+
+const BoothContainer = styled.div`
+    flex: 1;
+    display: flex;
+    align-items: center;
+    padding: 10px;
+    width: 100%;
+    justify-content: space-between;
+    border-bottom: 1px solid #CEDCEA;
+
+    img{
+        width: 100px;
+        height: 100px;
+    }
+`;
+
+const BoothContent = styled.div`
+    width: 35%;
+    height: 100px;
+    margin-left: 10px;
+    margin-right: 10px;
+    
+    p {
+        font-size: 17px;
+        margin-top: 6px;
+        margin-bottom: 6px;
+        color: #0047C9;
+        overflow: hidden;
+    }
+`;
+
+const BoothInfoBottom = styled.div`
+    color: #BBC7D3;
+    font-size: 16px;
+
+    div {
+      display: inline-block;
+      margin-top: 2.5px;
+      margin-bottom: 2.5px;
+      margin-right: 5px;
+    }
+`;
+
+const LikedNumber = styled.div` 
+    padding-top: 2.5px;
+    padding-bottom: 2.5px;
+    padding-left: 10px;
+    padding-right: 10px;
+    display: flex;
+    background-color: #EBF2FF;
+    border-radius: 15px;
+    align-items: center;
+    justify-content: center;
+    color: #0047C9;
+  
+    span {
+      font-size: 13px;
+      margin: 1px;
+    }
+`;
+
+type BoothListProps = {
+  booths: Booth[];
+  selectedDay: string;
+};
+
+export default function BoothList({
+  booths,
+  selectedDay,
+}: BoothListProps) {
+  if (!booths) { return []; }
+  const formatter = new Intl.NumberFormat('en', { notation: 'compact' });
+
+  return (
+    <Container>
+      {booths.map((booth) => {
+        console.log(booth);
+
+        const {
+          id, name, category, liked, boothImgs,
+        } = booth;
+
+        const boothDay = booth.boothDays.filter((
+          booth,
+        ) => booth.day === selectedDay);
+
+        return (
+          <PreButton
+            to={`/map/${id}`}
+            key={id}
+          >
+            <BoothContainer>
+              <img src={boothImgs[0].url} alt="boothImg.png" />
```

Comment:
배열을 인덱스로 다루는 건 일반적으로는 좋지 않은데요.
이런식으로 처리해주는 편이에요~
```js
const [firstBoothImage] = boothImgs;

<img src={firstBoothImage.url} alt="boothImg.png" />
```

Reply:
img 태그의 alt 속성에는 파일 확장자는 제거해주는게 좋을 것 같아요

Reply:
그리고 지금과 같은 경우에는 데이터와 함께 표현해주면 좋아요.~
```suggestion
              <img src={boothImgs[0].url} alt=`${name} 부스` />
```